### ISSUE TITLE
Slow matrix download?

### ISSUE THREAD

**Issue by kellrott** [bmeg/grip#129]
Need to check the performance of 

```
c = O.query().V().where(aql.eq("_label", "Individual"))\
.where(aql.and_(aql.eq("source", "tcga"), aql.eq("disease_code", "BRCA")))\
.in_("sampleOf").mark("a").in_("expressionFor").render(["$a.name", "$.expressions"])
data = {}
for row in c:
    #print("Loading:", row[0])
    data[row[0]] = row[1]
samples = pandas.DataFrame(data).transpose().fillna(0.0)
```

**Comment by kellrott** [comment]
Based on some rough observations, it seems like a lot of the time is in the client side python code. Maybe think about trying another JSON parser ( https://artem.krylysov.com/blog/2015/09/29/benchmark-python-json-libraries/ )

**Comment by kellrott** [comment]
This ended up being a bad chunk size in the python http client, fixed in 8f7b49ca1166a23ecab2675a82cd8188431f2671
